src/bindings/c/tests/CMakeLists.txt

```diff
@@ -41,6 +43,44 @@ endif()
 add_cpplint_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME})
 
 install(TARGETS ${TARGET_NAME}
-        RUNTIME DESTINATION tests
-        COMPONENT tests
-        EXCLUDE_FROM_ALL)
+    RUNTIME DESTINATION tests
+    COMPONENT tests
+    EXCLUDE_FROM_ALL)
+
+# OpenVINO 2.0 C API test sample
+set(TARGET_NAME "openvino_capi_test")
```

Comment:
I think we can have a common binary file for all tests for new and old API.
Maybe we can rename legacy test binary and fix CI steps.

Reply:
yes, it is a good idea, we will create another PR for test cases.

Reply:
Why cannot we do it in this PR? 😄

Reply:
Is it better to make this PR focus on API definition, and the following test and document can be put into another PRs?

Reply:
In this PR we add functional, I think we also need to have tests for this functional. Will be better to have and review small files.

Reply:
ok, let's split test sample to small files.